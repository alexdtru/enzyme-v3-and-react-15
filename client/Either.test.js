import React from "react";
import { render } from "enzyme";

import Either from "./Either";

it("renders correctly Right", () => {
  const test = render(<Either when={true} right={<span>Right</span>} />);
  expect(test).toMatchSnapshot();
});

it("renders correctly Left", () => {
  const test = render(<Either when={false} left={<span>Right</span>} />);
  expect(test).toMatchSnapshot();
});
