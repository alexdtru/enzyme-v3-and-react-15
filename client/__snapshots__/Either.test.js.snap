// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Left 1`] = `
<span>
  Right
</span>
`;

exports[`renders correctly Right 1`] = `
<span>
  Right
</span>
`;
